lib/routes/receive_payment/onchain/liquid_address/receive_liquid_address_payment_page.dart

```diff
@@ -0,0 +1,315 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_breez_liquid/flutter_breez_liquid.dart';
+import 'package:l_breez/cubit/cubit.dart';
+import 'package:l_breez/routes/receive_payment/lightning/widgets/widgets.dart';
+import 'package:l_breez/routes/receive_payment/widgets/address_widget/address_qr.dart';
+import 'package:l_breez/theme/theme.dart';
+import 'package:l_breez/utils/exceptions.dart';
+import 'package:l_breez/utils/min_font_size.dart';
+import 'package:l_breez/utils/payment_validator.dart';
+import 'package:l_breez/widgets/amount_form_field/amount_form_field.dart';
+import 'package:l_breez/widgets/flushbar.dart';
+import 'package:l_breez/widgets/keyboard_done_action.dart';
+import 'package:l_breez/widgets/loader.dart';
+import 'package:l_breez/widgets/single_button_bottom_bar.dart';
+import 'package:l_breez/widgets/transparent_page_route.dart';
+import 'package:service_injector/service_injector.dart';
+import 'package:share_plus/share_plus.dart';
+
+class ReceiveLiquidAddressPaymentPage extends StatefulWidget {
+  static const routeName = "/receive_bitcoin_address";
```

Comment:
```suggestion
  static const routeName = "/receive_liquid_address";
```

Reply:
Removed `ReceiveLiquidAddressPaymentPage` as it's unused 56df30ea90ce63a171c86d312807e7b23d96db4f